survival-analysis/package_changes.Rmd

```diff
@@ -0,0 +1,53 @@
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Package changes for censored regression models
+
+These are notes from a session where we discussed the required changes to tidymodels. Some open issues are:
+
+* Where does the user specify the time points for certain performance metrics (e.g. ROC methods)? There could be an option to the `tune` functions for this or it could be encoded into a metric set. In either case, the `.time` vector will be passed to the `parsnip` predict calls (when appropriate) so that the right predictions are created. We will _not_ have a `.time` argument to `yardstick` metrics since the times will be in the results of the prediction functions. 
```

Comment:
Might want to include here that we said for tune, we were leaning toward this being handled in the `control_` functions, not an argument to the main `tune_` functions themselves.